x/feemarket/types/keys.go

```diff
@@ -12,14 +12,12 @@ const (
 	RouterKey = ModuleName
 )
 
-// prefix bytes for the EVM persistent store
+// prefix bytes for the feemarket persistent store
 const (
 	prefixBlockGasUsed = iota + 1
-	prefixBaseFee
 )
 
 // KVStore key prefixes
 var (
 	KeyPrefixBlockGasUsed = []byte{prefixBlockGasUsed}
-	KeyPrefixBaseFee      = []byte{prefixBaseFee}
```

Comment:
@thomas-nguy @yihuang I think this requires a migration of state -> param for you guys (we can address this on a separate PR)

Reply:
yeah it is needed after this PR
https://github.com/tharsis/ethermint/pull/943

this line is just code cleaning